tests/test_v5_client.py

```diff
@@ -1,10 +1,12 @@
 from luminoso_api.v5_client import LuminosoClient, get_root_url
 from luminoso_api.errors import (
-    LuminosoClientError, LuminosoServerError, LuminosoError
+    LuminosoClientError, LuminosoServerError, LuminosoError,
+    LuminosoTimeoutError
 )
 
 import pytest
 import requests
+import sys
```

Comment:
One last nitpick: this import looks extraneous.

Reply:
You are right - I know I needed it there at one point, but I guess that code got removed and the import didnt